docs/book/toc.md

```diff
@@ -12,8 +12,6 @@
   * [Deploy using HuggingFace Spaces](getting-started/deploying-zenml/deploy-using-huggingface-spaces.md)
   * [Deploy with custom images](getting-started/deploying-zenml/deploy-with-custom-image.md)
   * [Manage deployed services](getting-started/deploying-zenml/manage-the-deployed-services/README.md)
```

Comment:
What happens to this section now?

Reply:
i was thinking
- the troubleshoot stack components can go into the Manage stacks and componets section
- the secret store thing can go inside the secret management section.
wdyt?

Reply:
implemented this